### DISCUSSION TITLE
MW4 first use

### DISCUSSION THREAD

**Discussion by alanrock24** [mworion/MountWizzard4#268] [Q&A]
II ran a model with MW4 tonight and had all kinds of problems. 
1: It completely skipped over a few points in the model and I don't know why
2: when I was finished, the handset on my 1000 was flashing "Warning 27 Min Tracking" 
3: MY mount was parked and I couldn't unpark it. Also, my model had an RMS error over 87,000.
Where do I begin to look for problems?

**Comment by mworion** [comment]
Hi Alan,
that sounds strange. Could you just drop the log file here?
Michel

**Reply by Ruediger3** [reply]
Hi Michel, Hi Alan,

@mworion 

I am supporting Alan a bit with his problem via 10M Forum and have seen he has contacted you via GitHub too. That's a good idea.

For reference: https://www.10micron.com/forum/viewtopic.php?f=2&t=1810&start=30

I think the time warning is the meridian flip warning. Could be that MW has set other limits. 
For the not solved point the log is the right place for sure. 

Maybe we can help Alan to get the things sorted out. 

Rüdiger

**Comment by alanrock24** [comment]
On Fri, Aug 26, 2022 at 11:52 PM Michael ***@***.***> wrote:

> Hi Alan,
> that sounds strange. Could you just drop the log file here?
> Michel
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mworion/MountWizzard4/discussions/268#discussioncomment-3488156>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYXFLVQ7UWNA45EX6A7KGFTV3G3J7ANCNFSM57YYUG3Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

[image: width=]
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by alanrock24** [reply]
I emailed the logfile but I will attach it again here. When my modeling run started. my camera was failing to download images, so I paused the run, disconnected ad reconnected the camera, and then resumed. 
[mw4-2022-08-27.log](https://github.com/mworion/MountWizzard4/files/9438052/mw4-2022-08-27.log)
Also, I didn't see where to set the exposure times or the filter. Can you point me in that direction?
Thanks,
Alan

**Reply by alanrock24** [reply]
I found the imaging settings under Settings-->Imaging (right where they belong!!)
Also, I'm wondering if the fact that my camera had issues and I had to disconnect and reconnect it caused problems.
Ironically, I loaded an "old" model and the mount seemed to work ok with the older, "bad" model, which I didn't expect.
Is there anyway to look at the (new) model to see if there are inherent problems with it?

**Reply by mworion** [reply]
Hi Alan,
I had a look to the log file. So far no strange things in it. A not solve point in on low altitude, so obstruction might be the reason.
If some point do not solve, this should not cause any problem in the model. If the number is too large, the settings of the platesolver might be to tight (please increase the FOV in driver settings)
MW4 does not set different Slew Limits itself. But if you have set them much different from small values around meridian, some of the point the mount will try to slew them from the "wrong" side. You could remove build points, which are too close to your   meridian settings to avoid this behavior of the mount.
If you loaded an old model, this should work if you did not change physical setups. but in any case you could analyse an old model without loading it to the mount in the analyse windows by selecting the stored data (timestamps and model type in the name will help you).
@Rüdiger: Thanks for helping. It's always good if two look to help.
Michel

**Reply by alanrock24** [reply]
There were a couple of points that it didn't even try to solve, it just skipped them completely, and no, those points were not below the horizon-- they just got skipped completely. It was odd the the "old" model (created with the 10u NINA Plugin) worked since I had definitely disassembled and reassembled my setup in the interim. When I did a go -to with the "old" model, it was way off, but after one plate solve, the mount went exactly where it was supposed to! After that I ran a 2 minute sub which looked great, and then it got cloudy again. I'll try a smaller model next time and we will see what kind of results I get. I looked at the log and didn't see much helpful information there either.
Alan

**Reply by mworion** [reply]
Thanks for trying it. Still there might be some hints in the log if you set the log level to debug as I described in the other thread.
Michel

**Comment by alanrock24** [comment]
Thank you. I didn't want to bother you since it's not your product. I
figured I would reach out to Michel.
Alan

[image: width=]
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
